cmd/cluster.go

```diff
@@ -487,6 +495,39 @@ func (c *config) clusterCreate() error {
 		disablePodSecurityPolicies = pointer.Pointer(viper.GetBool("disable-pod-security-policies"))
 	}
 
+	var networkAccessType *string
+	if viper.IsSet("network-isolation") {
+		networkAccessType = pointer.Pointer(viper.GetString("network-isolation"))
+		switch *networkAccessType {
+		case models.V1ClusterCreateRequestNetworkAccessTypeForbidden:
+			fmt.Printf(`
+WARNING: You are going to create a cluster which has no internet access with the following consequences:
+- pulling images is only possible from private registries you provide, these registries must be resolvable from the public dns and must be secured with a trusted TLS certificate
+- service type loadbalancer can only be created in networks which are specified in the allowed networks (see cluster inputs)
+- cluster wide network policies can only be created in certain network ranges whichspecified in the allowed networks (see cluster inputs)
+- It is not possible to change this cluster back to %q after creation
+`, models.V1ClusterCreateRequestNetworkAccessTypeBaseline)
+			err := helper.Prompt("Are you sure? (y/n)", "y")
+			if err != nil {
+				return err
+			}
+		case models.V1ClusterCreateRequestNetworkAccessTypeRestricted:
+			fmt.Printf(`
+WARNING: You are going to create a cluster which has no internet access with the following consequences:
+- pulling images is only possible from private registries you provide, these registries must be resolvable from the public dns and must be secured with a trusted TLS certificate
+- you can create cluster wide network policies to the outside world without restrictions
+- but cluster wide network policies allowing access to popular registries like docker.io, quay.io and eu.gcr.io will still not allow pulling images from there
```

Comment:
not true, with 0.0.0.0/0 port 443 images can also be pulled from these registries
```suggestion
```

Reply:
already adopted this sentence